IG_step-by-step installation content review (#3675)
wazuh/wazuh-documentation: source/installation-guide/open-distro/all-in-one-deployment/all_in_one.rst

Step-by-step installation
=========================

Install Wazuh and Open Distro for Elasticsearch components in an all-in-one deployment. This type of deployment is appropriate for testing and small working environments.  

Follow the instructions to configure the official repositories to perform installations. As an alternative, the installation can also be done using packages. Check out the list of available packages in the :ref:`Packages list <packages>` section. 

.. note:: Root privileges are required to execute all the commands.

.. _all_in_one_elastic:

Prerequisites
-------------

> To perform the following tasks, Open Distro for Elasticsearch requires the Java Development Kit as well as the installation of other packages, such as ``wget``, ``curl``, ``unzip``, and ``libcap``.

Reviewer: ```suggestion
Java Development Kit is installed as it is required by Open Distro for Elasticsearch.  To perform the following tasks, install ``wget``, ``curl``, ``unzip``, and ``libcap`` packages:
```
